### PR TITLE
Add a new generator type and a new property to timestamp config

### PR DESCRIPTION
**Pull request by klxfeiyang**

### CODE REVIEW

**Comment by klxfeiyang** [comment]
cc @albertshau, @dli357

File: src/main/java/io/cdap/plugin/datagen/generator/RandomChosenStringGenerator.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/datagen/generator/RandomChosenStringGenerator.java]: 2023

File: pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>datagen-plugins</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>0.2.1-SNAPSHOT</version>
```
Comment (albertshau) [inline pom.xml]: it's already a snapshot version, so no need to bump it.

**Review by albertshau — COMMENTED** [review]
minor comments, otherwise lgtm

File: pom.xml (hunk 2 above)
Comment (klxfeiyang) [reply pom.xml]: reverted

File: src/main/java/io/cdap/plugin/datagen/generator/RandomChosenStringGenerator.java (hunk 1 above)
Comment (klxfeiyang) [reply src/main/java/io/cdap/plugin/datagen/generator/RandomChosenStringGenerator.java]: updated

**Comment by klxfeiyang** [comment]
hey @albertshau, thanks for the review.   resolved your comments.  PTAL

**Review by albertshau — APPROVED** [review]

**Pull request merged**